src/lib/__tests__/util-test.js

```diff
@@ -812,4 +786,12 @@ describe('lib/util', () => {
             expect(util.getProp(a, 'foo.bar', 'default')).to.equal('default');
         });
     });
+
+    describe('inBoxWebApp()', () => {
+        it('should return false when window location is not a Box domain', () => {
+            expect(util.inBoxWebApp()).to.be.false;
+        });
+
+        /* No quick way to stub window.location.hostname to test for truth case */
```

Comment:
According to SO this works for href, maybe will work for hostname too?
```
global.window = {
       location: {
           href: {
               value: 'foo'
           }
       }
   }
```

Reply:
I found that thread and wasted ~an hour yesterday trying to get the test to work unsuccessfully. Tried a few things including Object.defineProperty, global.window (which wouldn't work in the browser anyways I think), and a few others.